src/tapyrus-genesis.cpp

```diff
@@ -87,8 +87,63 @@ static int AppInit(int argc, char* argv[])
     return CONTINUE_EXECUTION;
 }
 
+static int generateNewKeyPair()
+{
+    //generate a secret key
+    CKey secret;
+    secret.MakeNewKey(true);//compressed
+
+    //generate its public key
+    CPubKey pubkey = secret.GetPubKey();
+    assert(pubkey.IsFullyValid());
+    assert(secret.VerifyPubKey(pubkey));
+
+    fprintf(stdout, "private key:%s\n", EncodeSecret(secret).c_str());
```

Comment:
```suggestion
    fprintf(stdout, "private key: %s\n", EncodeSecret(secret).c_str());
```

It is more convenient to copy if there is space.